biome.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "vcs": { "enabled": false, "clientKind": "git", "useIgnoreFile": false },
```

Comment:
We want this to be enabled and use the ignore files, otherwise it will try to lint a lot of useless files